tasks/fund-link.js

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Could we add a task to `withdrawLink` also?

Reply:
great suggestion! I've added it in.